### DISCUSSION TITLE
Casual Community Chat

### DISCUSSION THREAD

**Discussion by tylerforesthauser** [tw-in-js/twind#71] [General]
Hey guys! I am a recent adopter of Twind (converting from Twin.macro + Emotion) and I am a big fan of this project, its philosophy and just general, overall approach. 

I’m wondering, is there (or plans to start) a community server of some kind (Slack, Discord, Hangouts, etc)? It would be nice to have a more casual (and “real-time”) environment to pick the developer’s brains and interact with the community, as opposed to the discussions possible here. 

Thank you both, Luke and Sascha, for the work that’s gone into this project (and your previous iterations). One way or another, I look forward to watching this project grow.

**Comment by sastan** [comment]
Thank you. This means a lot.

We have currently not planned something like that as we are quite busy preparing a 1.0 release. I like the idea of real-time chat. Maybe we could start with a meeting of some kind and expand that to a regular session. @lukejacksonn What do you think?

**Reply by lukejacksonn** [reply]
I think that a discord and perhaps a twitter account is a good idea. The more we grow the more this might become necessary. Perhaps we should do this prior to V1.0 as to give people a better chance to feedback?

**Reply by sastan** [reply]
Sounds good to me.

**Reply by lukejacksonn** [reply]
Ok I have created a discord server just now here is the link https://discord.gg/Mz4pFeVK

**Reply by natedunn** [reply]
Thanks @lukejacksonn. looks like the link expired tho. might need to set no limit.

**Reply by sastan** [reply]
We have added a badge to the readme: https://discord.com/invite/2aP5NkszvD
